composer.json

```diff
@@ -36,7 +36,8 @@
     "allow-plugins": {
       "phpro/grumphp": true,
       "dealerdirect/phpcodesniffer-composer-installer": true,
-      "php-http/discovery": true
+      "php-http/discovery": true,
+      "cweagans/composer-patches": true
```

Comment:
Needed for `composer update --prefer-lowest`, as some older dependency pulls patches

Reply:
yes, its the openeuropa/codereview, add a not in the readme to remove this as soon as we drop it